docs/mg7-paper/paper.md

```diff
@@ -139,20 +139,19 @@ In the first step the paired-end reads, designed with an insert size that yields
-Once we have the definitive HSPs selected, we obtain their corresponding taxonomic nodes using the taxonomic assignments that NCBI provides for all the nt database sequences. Now we have to analyze the topological distribution of these nodes in the taxonomy tree: i. If all the nodes forms a line in the taxonomy tree (are located in a not branched lineage to the tree root) we should choose the most specific taxID as the final assignment for that read. We call to this kind of assignment the ‘in line’ exception (see Figure 2 right panel). ii. If not, we should search for the *sensu stricto* Lowest Common Ancestor (LCA) of all the selected taxonomic nodes (See Figure 2 left panel). In this approach we decided to use the bitscore for evaluating the similarity because it is a value that increases when similarity is higher and depends a lot on the length of the HSP.
-Some reads could not find sequences with enough similarity in the database and then they would be classified as reads with no hits.
-Advanced metagenomics analysis approaches [@huson2012microbial] have adopted LCA-based assignment algorithms because it provides fine and trusted taxonomical assignment.
+#### LCA Assignment
+
+The LCA assignment simply computes the lowest common ancestor of the set of valid hits. Other metagenomics analysis approaches [@huson2012microbial] have adopted LCA-based assignment algorithms, as it provides the most precise assignment for a set of taxa, with no further information available^[the lowest common ancestor of a set of nodes in a tree is just their product in the tree viewed as a poset, with the "parent of" partial order]; as our reference database is already heavily preprocessed and checked for internal consistency, there is no need for altering the lowest common ancestor in any way, as is customarily done in other cases<!-- TODO cite? -->. MG7 operates under the assumption that hits which fulfill the user-defined criteria for being valid hits should be treated on an equal footing, and in that case taking their LCA is the coherent assignment choice with minimum information loss.
```

Comment:
> As a member of the "unfamiliar with trees in mathematical context" set of readers I like this type of footnote but it would be useful to have a list of definitions of mathematical terms (in this case poset and partial order would be in the list)

completely agree on the need of a list of terms